@ecomplus/storefront-components/src/js/TheAccount.js

```diff
@@ -75,6 +75,10 @@ export default {
       }
     },
 
+    authorization () {
+      return this.ecomPassport.checkAuthorization()
+    },
+
```

Comment:
`this.ecomPassport.checkAuthorization()` não retorna um objeto reativo, não vai funcionar da forma que você imaginou aqui..

Reply:
Na verdade esse computed não faz diferença nenhuma, se você quer observar o login do customer precisa adicionar um:

```js
this.ecomPassport.on('login', () => {
  // ...
})
```

Reply:
Tipo aí https://github.com/ecomplus/storefront/blob/master/%40ecomplus/storefront-components/src/js/PointsApplier.js#L105-L108